### ISSUE TITLE
Quick Start results in an empty page

### ISSUE THREAD

**Issue by smazurov** [cedvan/docker-toran-proxy#8]
Running the **Quick Start** (changed the port to `8090`, otherwise same command), when I go to the http://192.168.10.10:8090, it produces a blank page. Note, `192.168.10.10` is my `boot2docker` ip and other containers work fine.

`docker logs toran-proxy` produces this:

```
Configure Toran Proxy...
Creating logs directories...
Installing Cron...
Loading Nginx vhosts...
Starting PHP-FPM...
Starting Cron...
Starting Nginx..
```

I don't have time to debug this, but I figured I'd bring it up.

**Comment by cedvan** [comment]
Hello, 

Please, share your docker command, I'll debug this ;). 

Thanks you

**Comment by smazurov** [comment]
its virtually the Quick Start command:

```
docker run --name toran-proxy -d \
    -p 8090:80 \
    cedvan/toran-proxy:1.1.7
```

**Comment by smazurov** [comment]
I just realized its probably useful to know the docker version:

```
Client version: 1.7.0
Client API version: 1.19
Go version (client): go1.4.2
Git commit (client): 0baf609
OS/Arch (client): darwin/amd64
Server version: 1.7.0
Server API version: 1.19
Go version (server): go1.4.2
Git commit (server): 0baf609
OS/Arch (server): linux/amd64
```

**Comment by cedvan** [comment]
Hi,

It was bug with token github empty, this is fixed to version 1.1.7-1.
Test new quick start please

**Comment by smazurov** [comment]
yup, that fixed it, thanks!
